### DISCUSSION TITLE
Open Allure report in a specified directory instead of temporary directory

### DISCUSSION THREAD

**Discussion by adeel-plugit** [allure-framework/allure2#2676] [General Discussion]
Hi all,
I'm using allure reports in my company test reports. 
First, I'm confused about whether I should use the allure open or the allure command line for publishing the allure test results. 
Secondly, If there's no difference and I use the allure serve command then how can I specify it to generate the allure report in a specified folder instead of creating a temporary directory? 
Basically, I want to copy the history folder from the Allure report to the Allure results folder. 

Thanks

**Comment by baev** [comment]
`allure serve` is a shortcut for `allure generate` and `allure open`
